edgedb/protocol/codecs/codecs.pyx

```diff
@@ -20,6 +20,7 @@
 import decimal
 import uuid
 import datetime
+from edgedb.datatypes.datatypes import RelativeDelta
```

Comment:
We usually import modules in python/cython code

Reply:
```suggestion
from edgedb.datatypes import datatypes
```

Reply:
changed